internal/cmd/ske/kubeconfig/create/create.go

```diff
@@ -0,0 +1,198 @@
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	ClusterName    string
+	KubeconfigPath *string
+	ExpirationTime *string
+}
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   fmt.Sprintf("create %s", clusterNameArg),
+		Short: "Creates a kubeconfig for an SKE cluster",
+		Long: fmt.Sprintf("%s\n%s",
+			"Creates a kubeconfig for a STACKIT Kubernetes Engine (SKE) cluster.",
+			"By default the kubeconfig is created in the .kube folder, in the user's home directory. The kubeconfig file will be overwritten if it already exists."),
+		Args: args.SingleArg(clusterNameArg, nil),
+		Example: examples.Build(
+			examples.NewExample(
+				`Create a kubeconfig for the SKE cluster with name "my-cluster"`,
+				"$ stackit ske kubeconfig create my-cluster"),
+			examples.NewExample(
+				`Create a kubeconfig for the SKE cluster with name "my-cluster" and set the expiration time to 30 days`,
+				"$ stackit ske kubeconfig create my-cluster --expiration 30d"),
+			examples.NewExample(
+				`Create a kubeconfig for the SKE cluster with name "my-cluster" and set the expiration time to 2 months`,
+				"$ stackit ske kubeconfig create my-cluster --expiration 2M"),
+			examples.NewExample(
+				`Create a kubeconfig for the SKE cluster with name "my-cluster" in a custom location`,
+				"$ stackit ske kubeconfig create my-cluster --location /path/to/config"),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			model, err := parseInput(cmd, args)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(cmd)
+			if err != nil {
+				return err
+			}
+
+			if !model.AssumeYes {
+				prompt := fmt.Sprintf("Are you sure you want to create a kubeconfig for SKE cluster %q? This will OVERWRITE your current configuration, if it exists.", model.ClusterName)
+				err = confirm.PromptForConfirmation(cmd, prompt)
+				if err != nil {
+					return err
+				}
+			}
+
+			// Call API
+			req := buildRequest(ctx, model, apiClient)
+			resp, err := req.Execute()
+			if err != nil {
+				return fmt.Errorf("create kubeconfig for SKE cluster: %w", err)
+			}
+
+			// Create a config file in $HOME/.kube/config
+			configPath := model.KubeconfigPath
+
+			if configPath == nil {
+				userHome, err := os.UserHomeDir()
+				if err != nil {
+					return fmt.Errorf("could not get user home directory: %w", err)
+				}
+
+				err = os.MkdirAll(fmt.Sprintf("%s/.kube", userHome), 0o700)
+				if err != nil {
+					return fmt.Errorf("could not create kube directory: %w", err)
+				}
+				configPath = utils.Ptr(fmt.Sprintf("%s/.kube", userHome))
+			}
+
+			err = os.WriteFile(fmt.Sprintf("%s/config", *configPath), []byte(*resp.Kubeconfig), 0o600)
+			if err != nil {
+				return fmt.Errorf("could not write kubeconfig file: %w", err)
+			}
+
+			fmt.Printf("Created kubeconfig file for cluster %s with expiration date %v.\n", model.ClusterName, *resp.ExpirationTimestamp)
```

Comment:
Do you think it would be relevant to also indicate where the kubeconfig was written to?